guide/14-deep-learning/how_retinanet_works.ipynb

```diff
@@ -0,0 +1,159 @@
+{
```

Comment:
<p>Comment: Above, you explained FPN. Now coming to retinaNet, we should provide some explanation...something that includes ( FPN head is added  resnet backbone..<a href="https://towardsdatascience.com/review-resnet-winner-of-ilsvrc-2015-image-classification-localization-detection-e39402bfa5d8" target="_blank"><strong>ResNet</strong></a><strong>+</strong><a href="https://towardsdatascience.com/review-fpn-feature-pyramid-network-object-detection-262fc7482610" target="_blank"><strong>FPN</strong></a><strong>&nbsp;as backbone&nbsp;</strong>for feature extraction,<strong>&nbsp;plus two task-specific subnetworks for classification and bounding box regression</strong>, forming the<strong>&nbsp;RetinaNet)</strong></p><p> </p><p>2-3 line overview of retinaNet will help.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/562/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Technically, FPN is formed using the backbone (ResNet). The backbone becomes a part of FPN. Then, we add two subnets.</p><p> </p><p>Yes, I can possibly add a couple of lines as an overview.</p>